packages/perspective-jupyterlab/src/config/plugin.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     entry: "./src/ts/index.ts",
     resolveLoader: {
         alias: {
```

Comment:
The inclination to resist scope creep is *much* appreciated - but if we're going to move these files at all, maybe we just move them to where they belong or leave them in `src/`?

Reply:
Ideally I'd like to be in a place we don't publish `/src` at the end of this PR, so I'm open to either 
1. Leave them where they are currently
2. Move them into `webpack-plugin` package 

Considering we're chatting about this, maybe it's best to move them into the `webpack-plugin` package?

Reply:
In doing some more work I've decided to just go with option 2 here.